### ISSUE TITLE
Import A Patch function not working (Mac OS Catalina 10.15.6)

### ISSUE THREAD

**Issue by bengersfood** [meanmedianmoge/zoia_lib#43]
Trying to import a patch opens a dialog and a patch can be selected, but then the patch does not show in the local repository when searched for by name afterward. 3 screenshots attached.

<img width="1763" alt="Screenshot 2020-07-26 19 16 23" src="https://user-images.githubusercontent.com/43231443/88486414-44c36200-cf75-11ea-8d65-13b9a92c7c85.png">

<img width="1760" alt="Screenshot 2020-07-26 19 16 45" src="https://user-images.githubusercontent.com/43231443/88486421-50168d80-cf75-11ea-8486-6b0e3a135b68.png">

<img width="1761" alt="Screenshot 2020-07-26 19 17 16" src="https://user-images.githubusercontent.com/43231443/88486430-56a50500-cf75-11ea-8241-8f08fc4b6e59.png">

**Comment by john-breton** [comment]
Thanks for the report. I just have a few follow-up questions :
1.  Does the patch show up at all in the Local Storage View? Or are you just unable to search for it?
2.  Did the patch already exist in the Local Storage View prior to importing?

**Comment by bengersfood** [comment]
I don’t believe Cuckoo’s patches are on Patch Storage. And all the local files on my system are patch storage downloads. So it’s no to both questions

**Comment by john-breton** [comment]
> I don’t believe Cuckoo’s patches are on Patch Storage. And all the local files on my system are patch storage downloads. So it’s no to both questions
> […](#)
> On 27 Jul 2020, at 13:11, John Breton ***@***.***> wrote: ﻿ Thanks for the report. I just have a few follow-up questions : Does the patch show up at all in the Local Storage View? Or are you just unable to search for it? Did the patch already exist in the Local Storage View prior to importing? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Perfect. I'll investigate this further.
